tests/robot-tests/tests/libs/create_emulator_release_files.py

```diff
@@ -162,37 +166,60 @@ def create_public_release_files(self):
             container_client = self.blob_service_client.create_container(
                 "downloads")
         except ResourceExistsError:
+            pass
+
             container_client = self.blob_service_client.get_container_client(
                 "downloads")
 
-        self.upload_files(container_client,
-                          self.all_files_zip_files)
-        self.upload_files(container_client,
-                          self.data_files)
+        try:
+            self.upload_files(container_client,
+                              self.all_files_zip_files)
+
+        except ResourceExistsError:
+            pass
+
+        try:
+            self.upload_files(container_client,
+                              self.data_files)
+
+        except ResourceExistsError:
+            pass
```

Comment:
There is already a try / except in `upload_files` itself, so I don't think these try / except are necessary.